chatkit-core/src/main/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomsStateDiffer.kt

```diff
@@ -0,0 +1,86 @@
+package com.pusher.chatkit.rooms.state
+
+import com.pusher.chatkit.state.Action
+import com.pusher.chatkit.state.JoinedRoom
+import com.pusher.chatkit.state.LeftRoom
+import com.pusher.chatkit.state.RoomUpdated
+import com.pusher.chatkit.state.State
+import org.reduxkotlin.GetState
+
+internal class JoinedRoomsStateDiffer(val stateGetter: GetState<State>) {
+
+    internal fun stateExists() = stateGetter().joinedRoomsState != null
```

Comment:
If a class is internal there's no need to mark its members as internal as that's just boilerplate.

Reply:
👍 have updated